packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+            button.dispatchEvent(new MouseEvent('mousedown', { bubbles: true }));
+            await nextTick();
+            button.dispatchEvent(new MouseEvent('mouseup', { bubbles: true }));
+            await nextTick();
+            button.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+            await longNextTick();
+            expect(devtools.classList.contains('closed')).to.equal(false);
+        });
+    });
+    describe('InspectorComponent', () => {
+        it('should change panel to "Inspector" with top button', async () => {
+            const button = devtools.querySelector('devtools-navbar button');
+            expect(button.textContent).to.equal('Inspector');
+            expect(button.classList.contains('selected')).to.equal(false, 'button is not selected');
+
+            button.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+            await longNextTick();
+
+            expect(button.classList.contains('selected')).to.equal(true, 'button is selected');
+            expect(!!devtools.querySelector('devtools-panel.active devtools-tree')).to.equal(true);
+        });
+        it('should display the VNode tab by default in info', async () => {
+            const button = devtools.querySelector('devtools-info devtools-navbar button');
+            expect(button.textContent).to.equal('VNode');
+            expect(button.classList.contains('selected')).to.equal(true, 'button is selected');
+        });
+        it('should display the root by default in info', async () => {
+            const about = devtools.querySelector('devtools-info .about');
+            const aResult =
+                '<div class="about">' +
+                '<span class="type">VNode</span> ROOT<span class="id">0</span>' +
+                '</div>';
+            expect(about.outerHTML).to.equal(aResult);
+
+            const properties = devtools.querySelector('devtools-info .properties');
+            const pResult =
+                '<div class="properties">' +
+                '<div class="divider">👤 About me</div>' +
+                '<table>' +
+                '<tbody>' +
+                '<tr><td>id</td><td>0</td></tr>' +
+                '<tr><td>type</td><td>ROOT</td></tr>' +
+                '<tr><td>length</td><td>4</td></tr>' +
+                '<tr><td>total length</td><td>22</td></tr>' +
+                '<tr><td>original tag</td><td>None</td></tr>' +
+                '<tr><td>text</td><td>"Titleabcdefghidiv"</td></tr>' +
+                '</tbody>' +
+                '</table>' +
+                '<div class="divider">👪 My Family</div>' +
+                '<table>' +
+                '<tbody>' +
+                '<tr><td>index</td><td>0</td></tr>' +
+                '<tr><td>children</td><td><ol><li>HEADING1</li><li>PARAGRAPH</li><li>PARAGRAPH</li><li>?</li></ol></td></tr>' +
+                '</tbody>' +
+                '</table>' +
+                '</div>';
```

Comment:
Same here, if I change *anything* in the DOM structure of the Devtools, this test will fail. I would prefer that the test tests that each property is properly displayed inside the div with class "properties" but nothing more, otherwise we won't be able to change anything in the display without having to adapt the tests. Granted, these tests are strong, but I fear they are in fact too strong. Not sure about that, we'll need to discuss it with the others.